src/open_inwoner/cms/cases/views/status.py

```diff
@@ -116,6 +120,23 @@ def get_context_data(self, **kwargs):
 
             statuses = fetch_status_history(self.case.url)
 
+            statustype_config_mapping = {
+                zaaktype_statustype.statustype_url: zaaktype_statustype
+                for zaaktype_statustype in ZaakTypeStatusTypeConfig.objects.all()
```

Comment:
Should be refactored in develop?

Reply:
@alextreme Do you mean there is an equivalent for this in Develop already?  These values are part of Steven's commits on which I am relying to build the front-end, so I need them :-)

Reply:
@jiromaykin I mean that the objects.all() call is incorrect/inefficient, but this should have already been fixed on develop. The values should remain the same afaik